src/GraphQL/Instrumentation/GraphQLTelemetryProvider.cs

```diff
@@ -0,0 +1,189 @@
+#if NET5_0_OR_GREATER
+
+using System.Diagnostics;
+using GraphQL.DI;
+using GraphQL.Execution;
+using GraphQL.Types;
+using GraphQL.Validation;
+using GraphQLParser.AST;
+
+namespace GraphQL.Instrumentation;
```

Comment:
Let's change namespace here and in `GraphQLTelemetryOptions` to `GraphQL.Telemetry` since _instrumentation_ has different meaning. As I can formulate, instrumentation is one of the possible approaches to add needed telemetry into the app. Also I would not mix terms with other ~.Instrumentation packages.

Reply:
As you wish.